doc/classes/Font.xml

```diff
@@ -136,7 +136,7 @@
 			<param index="0" name="char" type="int" />
 			<param index="1" name="font_size" type="int" />
 			<description>
-				Returns the size of a character, optionally taking kerning into account if the next character is provided.
+				Returns the size of a character. Does not take kerning into account.
```

Comment:
Not sure if mentioning kerning is even necessary anymore. I don't know _how_ any reader would assume kerning is accounted for.
Does it hurt though? No...?

Reply:
Fair point, I don't mind either way

Reply:
I think it's fine to be explicit.